addons/test_website/tests/test_views_during_module_operation.py

```diff
@@ -81,3 +83,137 @@ def test_01_cow_views_unlink_on_module_update(env):
         ('type', '=', 'qweb'),
         ('key', '=', 'test_website.update_module_child_view'),
     ]), "Specific COW views did not get removed!"
+
+
+@standalone('theme_views')
```

Comment:
@d-fence :wave: Another one.

It was discussed here if you remember (on top of our private messages) https://discord.com/channels/678381219515465750/704997651833159831/811181635806167051

Those** should really be in the testing suite, not only in nightly. We are starting to get more and more standalone tests in website, all testing very core/low level feature of website:
- either the multi-website mechanism
- or the theme install process
Quite import to say the least.

You asked me to bring this in guru meeting so you don't forget, but as I don't regularly comes to those meetings, I kinda forgot too.

** I get you might not be very inspired to add those tests on every build (even if I think it really should), but on staging it should be the bare minimum, to avoid merging PR breaking those behaviors.
Also, note that I am here speaking only about `standalone` tests not being the initial ones from ADT which are auto generated and tests operations for every module. It seems fair to keep those inside the nightly as they are heavy. I am speaking here about every standalone tests except those ones.

cc @Xavier-Do

Reply:
@Xavier-Do @d-fence Hello mates :wave: 
It will be a year in a few days since I am asking for that, both on PRs, in the runbot channel and in PM with @d-fence.
It was supposed to be "discussed?" in guru meeting but you said to AL "I am not too sure what RDE wants, I'll check, no need to talk it here" :shrug: :shrug: .

Can we move forward on this one please :-) ?

This is getting more and more urgent, we **only** put tricky/critical tests in standalone, which, if they break, means nothing will be working regarding theme install or multi website mechanism, which would be :boom:.

We really need to prevent merging something that would break those, and not only run it in the nightly (again, I am not speaking about auto generated ones, but the one we wrote).